src/feed-worker.js

```diff
@@ -19,7 +20,9 @@ exports.start = function() {
           postURL: post.link,
         };
         processedPosts.push(processedPost);
-        extractUrls.extract(post.description);
+
+        const arrayOfUrls = extractUrls.extract(post.description);
+        sortUrls.sortUrlsByType(arrayOfUrls, 'https://github.com');
```

Comment:
We already have code that does this in https://github.com/Seneca-CDOT/telescope/blob/master/src/github-url.js